src/components/common/SessionAccounts.vue

```diff
@@ -17,37 +16,30 @@
           <h3>{{ account.name }}</h3>
           <Bech32 :address="account.address" short-form />
         </div>
-        <TmBtn value="Go to Lunie" color="primary" />
       </div>
     </div>
+    <div class="button-add-account">
```

Comment:
This has no functionality yet, is that intended?

Reply:
It was intentionally left without functionality because it will use the functionality introduced in Jordans PR2798. I have commented it out for now.